waiter/src/waiter/shell_scheduler.clj

```diff
@@ -358,42 +358,62 @@
     instance))
 
 (defn- get-pid->memory
```

Comment:
We should have unit tests for this by mocking the response to the `sh/sh ps` calls.